docs/Actions/grant-user-role.md

```diff
@@ -0,0 +1,258 @@
+                        "Expression": "testpassword",
+                        "Value": "",
+                        "IsExpression": true
+                    },
+                    "RandomPass": "",
+                    "EmailField": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false
+                    },
+                    "FirstName": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false
+                    },
+                    "LastName": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false
+                    },
+                    "SendDnnMail": "",
+                    "LoginIfExists": "",
+                    "Authorization": {
+                        "Expression": "",
+                        "Value": "Authorize",
+                        "IsExpression": false,
+                        "Parameters": {}
+                    }
+                },
+                "ActionType": "UserManagement_UserRegistration",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "User Management"
+                    }
+                }
+            },
+            {
+                "Id": -1,
+                "$_uid": "action15918671753464006",
+                "Parameters": {
+                    "RoleId": {
+                        "Expression": "",
+                        "Value": "6",
+                        "IsExpression": false,
+                        "Parameters": {}
+                    },
+                    "RoleNames": "",
+                    "DateSelectionMode": {
+                        "Expression": "",
+                        "IsExpression": false,
+                        "Parameters": {},
+                        "Value": "DateWithOffset"
+                    },
+                    "ExtensionDays": "",
+                    "StartDate": {
+                        "Date": ""
+                    },
+                    "StartDateToken": "",
+                    "ExpireDate": {
+                        "Date": ""
+                    },
+                    "ExpireDateToken": "",
+                    "RoleExpiration": "30"
+                },
+                "ActionType": "GrantUserRole",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "User"
+                    }
+                }
+            }
+        ]
+    }
+}
+```
+
+## `Frequently asked questions`
```

Comment:
Remove the FAQ section and move the content in other sections where it falls in naturally.

Reply:
Done